medusa/providers/generic_provider.py

```diff
@@ -111,6 +112,27 @@ def kind(cls):
         """Return the name of the current class."""
         return cls.__name__
 
+    def create_magnet(self, filename, result):
+        """
+        Create a .magnet file containing the Magnet URI.
+
+        :param filename: base filename (without extension).
+        :param result: SearchResult object.
+        :returns: True if the magnet file is created and verified.
+        """
+        filename_ext = '{filename}.magnet'.format(filename=filename)
+        log.info('Saving magnet file {result} to {location}',
+                 {'result': result.name, 'location': filename_ext})
+        with io.open(filename_ext, 'wb') as fp:
```

Comment:
Is this really needed to be written as bytes? 
Also we can use open since we don"t need Python 2 compatibility